### ISSUE TITLE
vue3 tooling/example builds a broken app

### ISSUE THREAD

**Issue by adq** [single-spa/vue-cli-plugin-single-spa#25]
**TL;DR**
When you create a single-spa vue3 app with the create-single-spa tooling, it adds the following props in the top level render function: name. mountParcel, singleSpa.

It does not define these as props in the App.vue.

Therefore vue assumes they're attrs for the DOM element being rendered, attempts to add them and falls over because singleSpa is an object and thus cannot be converted to a simple string. Even if it did work, you get stray cruft attributes in the DOM.

Suggestion: remove these from the Vue3 default tooling/example doc as I would say these are props are for advanced usage. The Vue2 tooling/example does not supply them. 

You already have a separate section discussing them: https://single-spa.js.org/docs/ecosystem-vue/#custom-props-1 if someone needed to know how to do this.

**Original investigation follows**

Hi, I'm very new to single-spa-vue, but I think I've maybe spotted an issue with the vue3 example **and the create-single-spa tooling**.

On https://single-spa.js.org/docs/ecosystem-vue/, it says this:

```
[....]
const vueLifecycles = singleSpaVue({
  createApp,
  appOptions: {
    render() {
      return h(App, {
        // single-spa props are available on the "this" object. Forward them to your component as needed.
        // https://single-spa.js.org/docs/building-applications#lifecyle-props
        name: this.name,
        mountParcel: this.mountParcel,
        singleSpa: this.singleSpa,
      });
    },
  },
  handleInstance: (app) => {
    app.use(router);
  }
});
[...]
```

However, if I do that, I get the following failure when it tries to mount the component:
```
app-errors.js:12 Uncaught TypeError: application '@tvsquared/tvs-spapoc-nav' died in status SKIP_BECAUSE_BROKEN: Cannot convert object to primitive value
    at patchAttr (runtime-dom.esm-bundler.js?830f:168)
    at patchProp (runtime-dom.esm-bundler.js?830f:354)
    at mountElement (runtime-core.esm-bundler.js?5c40:3826)
    at processElement (runtime-core.esm-bundler.js?5c40:3789)
    at patch (runtime-core.esm-bundler.js?5c40:3709)
    at componentEffect (runtime-core.esm-bundler.js?5c40:4211)
    at reactiveEffect (reactivity.esm-bundler.js?a1e9:42)
    at effect (reactivity.esm-bundler.js?a1e9:17)
    at setupRenderEffect (runtime-core.esm-bundler.js?5c40:4176)
    at mountComponent (runtime-core.esm-bundler.js?5c40:4134)
```

Digging in shows that patchAttr is trying to set the values name, mountParcel, and singleSpa on the DOM element, which it can't do 'cos singleSpa is an object so cannot be converted to a primitive value.

I'm not sure what the correct way to fix this is; if I omit 
```
        name: this.name,
        mountParcel: this.mountParcel,
        singleSpa: this.singleSpa,
```

completely, things work, but I don't know what effect that will have.

Oh, I'm using vue 3.0.5, single-spa-vue 2.2.0 - basically I set it all up yesterday and ran into this problem.

Vue 2 works absolutely fine. I notice it does not pass those three props in the docs.

**Comment by adq** [comment]
Hey, no problem, just done it; see what you think.

**Comment by joeldenning** [comment]
Resolved in #26 and released in https://github.com/single-spa/vue-cli-plugin-single-spa/releases/tag/v2.1.1
